adafruit_display_shapes/triangle.py

```diff
@@ -129,13 +153,13 @@ def _draw_filled(self, x0, y0, x1, y1, x2, y2):
     # pylint: enable=invalid-name, too-many-locals, too-many-branches
 
     @property
-    def fill(self):
+    def fill(self) -> int:
```

Comment:
another pair here that can be `Optional[int]` for the return here, and it's argument type in the setter.